### ISSUE TITLE
Factor shared lambda helper functions into a helper file to reduce redundancy

### ISSUE THREAD

**Issue by olsonadr** [responsible-ai-collaborative/nlp-lambdas#17]
A lot of code will be re-used between lambdas so might as well factor out some of this code for future development. If it makes sense I might do this before starting on #15 so that they can use these.

I'm imagining some amount of config could be done per-lambda (maybe in a config file adjacent to each lambda handler script) so you can tell the Dockerfile which files to parcel into their containers, as not all future lambdas would need a copy of the model, all the helper functions, etc.

Common theme: I haven't done this yet to make sure we're feature-complete by Fri, but it pained me a little to ignore it

**Comment by olsonadr** [comment]
Now that I'm thinking about it, it might be nice to have a single config file in the `inference` directory that describes each lambda by giving it something like:
```json
{
"lambdas": [
  {
    "name": "text-to-db-similar",
    "handler_script": "text-to-db-similar.py",
    "required_resources": [
      "model/*",
      "helpers/longformer_helpers.py"
    ]
  },
  {
    "name": "text-to-db-similar",
    "handler_script": "text-to-db-similar.py",
    "required_resources": [
      "model/*",
      "helpers/longformer_helpers.py"
    ]
  }
]
}
```
where each path is a relative path starting from the `inference` directory
